procedural/scheduling/src/main/kotlin/gov/nasa/ammos/aerie/procedural/scheduling/SchedulingProcedureMapper.kt

```diff
@@ -3,7 +3,7 @@ package gov.nasa.ammos.aerie.procedural.scheduling
 import gov.nasa.jpl.aerie.merlin.protocol.types.SerializedValue
 import gov.nasa.jpl.aerie.merlin.protocol.types.ValueSchema
 
-interface ProcedureMapper<T: Goal> {
+interface SchedulingProcedureMapper<T: Goal> {
```

Comment:
Heads up: this rename will break existing scheduling goals. I actually encountered the reverse of this, where I built a goal on this branch and tried to use it on the db support branch, only to get `implementation not found` exception.